Quantum-Control-Applications/Superconducting/Single Flux Tunable Transmon/SSB mixer for readout/Use Case 1 - Paraoanu Lab - Cryoscope/readme.md

```diff
@@ -0,0 +1,221 @@
+# Cryoscope
+
+*Demonstrated on the experiment of Marko Kuzmanovic and Aidar Sultanov in the Lab of Prof. Sorin Paraoanu in Aalto University, 
+only two days after connecting the OPX to the quantum device.*
+
+*Important note: the code in this folder is the exact code that was used for running this measurement*
+*and it is tailored for a very specific setup and software environment. Thus, the code is here only for insipiration.*
+
+The goal of this use-case is to implement a cryoscope measurement as described in [[1]](#1). 
+The idea of the method is to measure the step response of the flux line using the qubit phase measured in a Ramsey-like experiment.
+The possibly distorted response can then be pre-corrected by designing suitable IIR and FIR filters.
+
+
+## 1. Experimental set-up and required calibrations
+
+
+<img align="right" src="Single Flux Tunable Transmon Setup.PNG" alt="drawing" width="400"/>
+
+### 1.1 Experimental set-up
+A standard flux tunable transmon coupled to a readout resonator set-up is used and the general scheme is represented on the figure on the right.

```

Comment:
What is a non-standard flux tunable transmon? maybe simple to say what it was: a transmon with a SQUID loop as nonlinear inductor with a flux line next to it? I'm not sure what was the device specifications.